process_report/process_report.py

```diff
@@ -331,7 +412,19 @@ def export_lenovo(dataframe: pandas.DataFrame, output_file=None):
     lenovo_df.rename(columns={SU_HOURS_FIELD: "SU Hours"}, inplace=True)
     lenovo_df.insert(len(lenovo_df.columns), "SU Charge", SU_CHARGE_MULTIPLIER)
     lenovo_df["Charge"] = lenovo_df["SU Hours"] * lenovo_df["SU Charge"]
-    lenovo_df.to_csv(lenovo_file_name)
+    lenovo_df.to_csv(output_file)
+    # TODO (Quan Pham) Where to place these
+
+
+def upload_to_s3(invoice_list: list, invoice_month):
+    invoice_bucket = get_invoice_bucket()
+    for invoice_filename in invoice_list:
+        invoice_b2_path = (
+            f"Invoices/{invoice_month}/{invoice_filename} {invoice_month}.csv"
```

Comment:
The output filenames already will contain an extension when provided as arguments from the command line, therefore you are double appending a `.csv` when you write to file and when you write them to `s3`.